### PR TITLE
Front end

### PR DESCRIPTION
**Pull request by farhanhaider1**
HTML and CSS for left semi-circle

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+body {		
+    margin: 0em;		
+    padding: 0em;		
+}
```
Comment (farhanhaider1) [inline style.css:4]: this will reset all existing padding

Comment (Tenzilla) [reply style.css:4]: Did you mean style.css will reset all existing padding?

File: index.html [hunk 2]
```diff
@@ -2,10 +2,15 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <meta name="HealthHq" content="width=device-width, initial-scale=1.0">
```
Comment (Tenzilla) [inline index.html:5]: Should we do "HealthHQ" or "HealthHq"?

**Review by Tenzilla — CHANGES_REQUESTED** [review]
We could change a little

File: style.css (hunk 1 above)
Comment (farhanhaider1) [reply style.css:4]: yeah sometimes a stupid white space will appear on top of the header! This will take care of it...

File: index.html (hunk 2 above)
Comment (farhanhaider1) [reply index.html:5]: umm i think HQ will be better

**Comment by farhanhaider1** [comment]
update to HealthHQ
The circle is ok!

**Pull request merged**